### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by kkirby**
My attempt to apply a standard to comments and clean up tabs/spaces. It's very possible that I took the wrong direction, though. Let me know if you want things changed.

### CODE REVIEW

**Comment by kkirby** [comment]
Forgot to mention, there were instances of `rct: address', I assumed this was a typo and `rct2:` was meant.

File: src/windows/sign.c [hunk 1]
```diff
@@ -267,7 +270,10 @@ static void window_sign_mouseup(rct_window *w, int widgetIndex)
 	}
 }
 
-/* rct2: 0x6B9784 & 0x6E6164 */
+/**
+ *
+ *  rct2: 0x6B9784
```
Comment (duncanspumpkin) [inline src/windows/sign.c]: Can you change this to the two addresses separated by a comma to match other functions that are based on multiple functions

**Comment by duncanspumpkin** [comment]
Good work, Have you done this with a script? The vehicle update code that will be ready for merging soon I know does not have the comments quite correct so just wanting to know if us with fixing manually.

**Comment by kkirby** [comment]
I've updated the branch. It was a combination of manual and automatic. I mainly just did a lot of find & replace with regular expressions. Some thing I used a script, though. I'm fairly sure there are still some lingering comments that don't follow this format, but I think having the majority updated will give future contributors an idea of the styling format. As for your new code, I'm sure it won't be a hassle to update if it needs to be.

**Pull request merged**